R/dependencies.R

```diff
@@ -540,25 +540,35 @@ renv_dependencies_discover_description <- function(path,
       renv_restore_state(key = "root") %||%
       renv_project_resolve()
 
-    # get fields from settings
+    # if this appears to be the DESCRIPTION associated with the active project,
```

Comment:
IMO it's time to refactor this big `{}` block because it's getting hard to see the overall flow of this function. Maybe we need `renv_dependencies_discover_description_fields()`?